supervisor/homeassistant/const.py

```diff
@@ -6,7 +6,7 @@
 
 from ..const import CoreState
 
-LANDINGPAGE: AwesomeVersion = AwesomeVersion("landingpage")
+LANDINGPAGE: AwesomeVersion = AwesomeVersion("latest")
```

Comment:
This change seems incorrect, this is meant to pick up the landing page tag.

Could you elaborate on the issue you are trying to fix?

Reply:
"landingpage" tag do not exist, so the docker image cannot be downloaded.
Issue #4796

This problem started appearing in issue #335

I also can't find any "landingpage" in any other repository, but there is always "latest", so why not start using "latest" and in place of "landingpage"?

Reply:
For the machines which are supported by Supervisor, the `landingpage` does exist. This would break user expierence on supported platform. I made some additional comments on https://github.com/home-assistant/supervisor/issues/4796#issuecomment-1913148213.